### DISCUSSION TITLE
Any way to cancel theme styling for subsets of the code?

### DISCUSSION THREAD

**Discussion by npolyak** [AvaloniaUI/Avalonia#6154] [Q&A] [answered]
For example, if a theme is defined at the application level but I want to have a special style for a button, is there any way to cancel all the styles for buttons throughout the application? Otherwise the resulting style setters will be the union of the setters from the theme, and my custom setters.

**Comment by maxkatz6** [comment, marked as answer]
No, it's not possible. 
If you are working on your own theme, you can define style classes for default buttons and special cases. 
But in general there was discussion about such a feature here https://github.com/AvaloniaUI/Avalonia/issues/2769

**Reply by npolyak** [reply]
Thanks Max
